### PR TITLE
Move RegionalOffice hashes to constants

### PR DESCRIPTION
**Pull request by lowellrex**
Related to #11349 where we want to be able to map from the regional office key (e.g. RO34) to the name of the city on the front-end without having to pass that information from the back-end.

### CODE REVIEW

File: app/models/regional_office.rb [hunk 1]
```diff
@@ -1415,14 +117,5 @@ def ro_facility_ids_for_state(state_code)
         ro[:facility_locator_id].present? && state_code == ro[:state]
       end.pluck(:facility_locator_id).uniq
     end
-
-    def find_ro_by_facility_id(facility_id)
```
Comment (lowellrex) [inline app/models/regional_office.rb]: We don't use this function anywhere so I removed it.

File: app/models/regional_office.rb [hunk 2]
```diff
@@ -1389,7 +91,7 @@ def ros_with_hearings
 
     # Returns RegionalOffice objects for each RO that has the passed station code
     def for_station(station_key)
-      [STATIONS[station_key]].flatten.map do |regional_office_key|
+      [STATIONS[station_key.to_sym]].flatten.map do |regional_office_key|
```
Comment (lowellrex) [inline app/models/regional_office.rb]: This type casting feels a little hacky but I am struggling to think of a better way to handle this. Open to suggestions!

Comment (pkarman) [reply app/models/regional_office.rb]: I find your approach readable and can't think of a better one. Strings and symbols in Ruby are notoriously annoying.

File: app/models/regional_office.rb [hunk 3]
```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
-# rubocop:disable Metrics/ClassLength
+def convert_top_level_key_to_string(hash)
+  hash.keys.map { |key| [key.to_s, hash[key]] }.to_h
+end
```
Comment (lowellrex) [inline app/models/regional_office.rb]: These hashes were previously formatted with the outermost key as a string, and the nested hashes' keys as symbols. This functions transforms the input consistently formatted nested hashes to match the previous formatting.

File: app/models/regional_office.rb (hunk 1 above)
Comment (lomaxap) [reply app/models/regional_office.rb]: ☝️ I'll be using this in a forthcoming PR

Comment (lowellrex) [reply app/models/regional_office.rb]: Ahh gotcha. Putting it back.

**Pull request merged**